src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/utils/functions.ts

```diff
@@ -267,7 +267,7 @@ export function getHighestCertificationType(options: CertificationComparison[]):
     throw new Error("No valid certification found in options");
   }
 
-  return parsed.reduce((best, curr) => (certificationWeights[curr] > certificationWeights[best] ? curr : best), parsed[0]);
+  return parsed.reduce((best, curr) => (certificationWeights[curr] > certificationWeights[best!] ? curr : best), parsed[0]);
```

Comment:
odd typescript issue, where it thinks best might be undefined, but we know it won't be undefined since the parsed length is greater than 0.